pxr/base/tf/stringUtils.h

```diff
@@ -572,6 +572,22 @@ TF_API std::string TfStringify(std::string const&);
 TF_API std::string TfStringify(float);
 /// \overload
 TF_API std::string TfStringify(double);
+/// \overload
+TF_API std::string TfStringify(short);
+/// \overload
+TF_API std::string TfStringify(unsigned short);
+/// \overload
+TF_API std::string TfStringify(int);
+/// \overload
+TF_API std::string TfStringify(unsigned int);
+/// \overload
+TF_API std::string TfStringify(long);
+/// \overload
+TF_API std::string TfStringify(unsigned long);
+/// \overload
+TF_API std::string TfStringify(long long);
+/// \overload
+TF_API std::string TfStringify(unsigned long long);
```

Comment:
Instead of using std::to_chars, could we just make the `ostringstream` in the base `TfStringify` implementation always use the classic locale?

```
std::ostringstream stream;
stream.imbue(std::locale::classic());
...
```

This is simpler and ensures other types going through this code path will also pick up this behavior. Also, our performance tests for layer writing (surprisingly?) showed negligible effects using `std::to_chars` but noticeable improvements using the classic locale.